lib/FindLocation/FindLocationLookup.js

```diff
@@ -135,13 +134,16 @@ export const FindLocationLookup = (props) => {
     libraries,
   ]);
 
-  const renderBeforeSearch = useCallback(() => {
+  const renderBeforeSearch = useCallback((resetFilters) => {
     return crossTenant
       ? (
         <AffiliationsSelection
           affiliations={affiliations}
           label={affiliationsLabel}
-          onChange={onTenantChange}
+          onChange={(tenant) => {
+            onTenantChange?.(tenant);
+            resetFilters?.();
+          }}
```

Comment:
what about the parent filters? https://github.com/folio-org/stripes-acq-components/pull/799/files#diff-baf096ec8293f7e616013ac8bb89d7f5518d788f82e5e40ae814fb184f2eeef0R54